pages/deploy/deploy_aws.md

```diff
@@ -531,34 +540,40 @@ check the [UserGuide](https://github.com/data-dot-all/dataall/blob/main/UserGuid
 
 
 ### Disable module features
-As you probably noticed, the `s3_datasets` module contains an additional field called `features` in the `config.json`. 
+As you probably noticed, the `s3_datasets` and `datasets_base` modules contain an additional field called `features` in the `config.json`. 
 If there is a particular functionality that you want to enable or disable you can do so in this section. 
 In the example config.json, the feature that enables file upload from data.all UI has been disabled.
 
```

Comment:
This is just an example to show how features work. We do not need to include the whole s3_datasets and datasets_base config. I think with s3_datasets is enough, have a read to see if the example is clear